src/Uno.Extensions.Serialization/IStreamSerializer.cs

```diff
@@ -37,3 +37,9 @@ public interface IStreamSerializer
     //     object or not.
     void WriteToStream(Stream stream, object value, Type valueType);
 }
+
+public interface IStreamSerializer<T> : IStreamSerializer
+{
+	T? ReadFromStream(Stream source);
```

Comment:
Shouldnt this be symmetrical with the string version and be called FromStream and ToStream?